pkg/chartrenderer/default.go

```diff
@@ -89,43 +93,37 @@ func (r *chartRenderer) RenderEmbeddedFS(embeddedFS embed.FS, chartPath, release
 	return r.renderRelease(chart, releaseName, namespace, values)
 }
 
-func (r *chartRenderer) renderRelease(chart *chartapi.Chart, releaseName, namespace string, values interface{}) (*RenderedChart, error) {
-	chartName := chart.GetMetadata().GetName()
-
+func (r *chartRenderer) renderRelease(chart *helmchart.Chart, releaseName, namespace string, values interface{}) (*RenderedChart, error) {
 	parsedValues, err := json.Marshal(values)
 	if err != nil {
-		return nil, fmt.Errorf("can't parse variables for chart %s: ,%s", chartName, err)
+		return nil, fmt.Errorf("failed to parse values for chart %s: ,%w", chart.Metadata.Name, err)
```

Comment:
Nit: my suggestion in the previous review was without the `,` 😅 

```suggestion
		return nil, fmt.Errorf("failed to parse values for chart %s: %w", chart.Metadata.Name, err)
```